ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Main.java

```diff
@@ -27,29 +36,90 @@
+            // create and add console appender
+            final PatternLayoutEncoder consoleEncoder = new PatternLayoutEncoder();
+            consoleEncoder.setContext(loggerContext);
+            consoleEncoder.setPattern(LOG_FORMAT_PATTERN);
+            consoleEncoder.start();
+
+            final ConsoleAppender<ILoggingEvent> consoleAppender = new ConsoleAppender<ILoggingEvent>();
+            consoleAppender.setContext(loggerContext);
+            consoleAppender.setName("STDOUT");
+            consoleAppender.setEncoder(consoleEncoder);
+
+            final ch.qos.logback.classic.filter.ThresholdFilter consoleFilter = new ThresholdFilter();
+            consoleFilter.setLevel(consoleLevel);
+            consoleFilter.setName(consoleLevel);
+            consoleFilter.start();
+            consoleAppender.addFilter(consoleFilter);
+
+            consoleAppender.start();
+            LOG.addAppender(consoleAppender);
+        }
+
+        if (!fileLevel.equalsIgnoreCase(Level.OFF.toString())) {
+            // create file appender only if needed.
+            // if done in the xml, it will create an empty file
+
+            final RollingFileAppender<ILoggingEvent> fileAppender = new RollingFileAppender<ILoggingEvent>();
+            final TimeBasedRollingPolicy<ILoggingEvent> timeBasedRollingPolicy = new TimeBasedRollingPolicy<ILoggingEvent>();
+            final DefaultTimeBasedFileNamingAndTriggeringPolicy<ILoggingEvent> timeBasedTriggeringPolicy = new DefaultTimeBasedFileNamingAndTriggeringPolicy<ILoggingEvent>();
+
+            fileAppender.setContext(loggerContext);
+            timeBasedTriggeringPolicy.setContext(loggerContext);
+            timeBasedRollingPolicy.setContext(loggerContext);
+
+            fileAppender.setRollingPolicy(timeBasedRollingPolicy);
+            timeBasedRollingPolicy.setParent(fileAppender);
+            timeBasedRollingPolicy.setTimeBasedFileNamingAndTriggeringPolicy(timeBasedTriggeringPolicy);
+            timeBasedTriggeringPolicy.setTimeBasedRollingPolicy(timeBasedRollingPolicy);
+
+            timeBasedRollingPolicy.setFileNamePattern(LOG_DIR + LOG_ARCHIVE_FILE_PATTERN);
+            timeBasedRollingPolicy.start();
+
+            timeBasedTriggeringPolicy.setCurrentTime(new Date().getTime());
+            timeBasedTriggeringPolicy.start();
+
+            final PatternLayoutEncoder fileEncoder = new PatternLayoutEncoder();
+            fileEncoder.setContext(loggerContext);
+            fileEncoder.setPattern(LOG_FORMAT_PATTERN);
+            fileEncoder.start();
+
+            fileAppender.setTriggeringPolicy(timeBasedTriggeringPolicy);
+            fileAppender.setEncoder((Encoder<ILoggingEvent>)fileEncoder);
+            fileAppender.setFile(LOG_DIR  +  LOG_FILE_NAME);
+            fileAppender.setName("LOGFILE");
+            timeBasedRollingPolicy.start();
+
+            final ch.qos.logback.classic.filter.ThresholdFilter fileFilter = new ThresholdFilter();
```

Comment:
Same question here about fully qualified name on the left, but not on the right.